spectator-ext-spark/src/test/java/com/netflix/spectator/spark/SparkNameFunctionTest.java

```diff
@@ -41,32 +41,29 @@ private void assertEquals(Id expected, Id actual) {
   //EXECUTOR
   @Test
   public void executorMetric() {
```

Comment:
Please keep the existing tests and add additional ones for all valid Spark 2.+ `name` values.

Reply:
Done